javascript/javascript3/week2/index.css

```diff
@@ -0,0 +1,73 @@
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin: 0;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: darksalmon;
+  background-image: url("./img/Chicawoof!\ -\ Dox\ Design\ -\ Our\ Clients.jpeg");
```

Comment:
Just to double check, does this work (load the background image correctly)?
To me it seems like the backslashes are not necessary.